doc/Customization-guide/How-to-contribute/Git-flow/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: Git flow
+permalink: /:path/
+nav: /:path/Customization-guide/How-to-contribute/Git-flow
+sort_idx: 10
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+We use a [GitHub flow](https://guides.github.com/introduction/flow/). That means, you should branch from the main repo and contribute back via pull requests. That's de facto standard workflow on GitHub—nothing fancy here.
```

Comment:
"That's de facto standard workflow on GitHub—nothing fancy here."
Can this be changed to just
"That is de facto standard work flow on GitHub with exception(s) described below."
?